src/components/SlateEditor/plugins/blockPicker/actions.tsx

```diff
@@ -141,6 +142,11 @@ export const commonActions: Action[] = [
     icon: <Globe />,
     helpIcon: renderArticleInModal("Gloss"),
   },
+  {
+    data: { type: TYPE_DISCLAIMER, object: "disclaimer" },
+    icon: <WarningOutline />,
+    helpIcon: renderArticleInModal("Disclaimer"),
+  },
```

Comment:
Kun admin skal ha tilgang til dette i følge NDLA
```suggestion
  {
    data: { type: TYPE_DISCLAIMER, object: "disclaimer" },
    icon: <WarningOutline />,
    helpIcon: renderArticleInModal("Disclaimer"),
    requiredScope: DRAFT_ADMIN_SCOPE,
  },
```